test/system/250-systemd.bats

```diff
@@ -314,9 +315,11 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     # stop systemd container
     service_cleanup
 
+    pasta_iface = default_ifname
```

Comment:
This makes the tests pass:
```diff
-    pasta_iface = default_ifname
+    pasta_iface=$(default_ifname)     <<<< remove the spaces. Add dollar-parens.
```
(Whether or not this results in a valid test, I don't know; haven't reviewed this yet).

From a quick glance, I'm not seeing what value is added from a separate `helpers.pasta`. Could those two functions be rolled into `.network` ?